catalogue/webapp/services/catalogue/works.js

```diff
@@ -79,7 +79,9 @@ export async function getWork({
   const params = {
     include: [
       ...workIncludes,
-      ...(toggles && toggles.archivesPrototype ? ['collection'] : []),
+      ...(toggles && toggles.archivesPrototype
```

Comment:
Can we not just put this in?

Reply:
If we're happy with the performance of the API with these, then yeah

Reply:
@warrd would you be happy for this?

Reply:
I think we should move as much as possible to the new API if that's what you are asking? I would like to remove the old API code when ready.

Reply:
@warrd it's more about if we're happy for these includes (parts, partOf etc.) to go in by default for all users, in terms of API performance

Reply:
I don't know if the new API is slower than the old one, is it that there is more separate requests using this method? I am happy if other people are.

Reply:
👍